File: 1980_find_unique_binary_string.java
import java.util.*;

class Solution {
    public String findDifferentBinaryString(String[] nums) {
        Set<Integer> t = new HashSet<>();
        for (String x : nums) t.add(Integer.parseInt(x, 2));
        for (int k = 0; k <= nums.length; k++) 
            if (!t.contains(k)) return String.format("%" + nums.length + "s", Integer.toBinaryString(k)).replace(' ', '0');
        return "";
    }
}

File: Leetcode_POTD_2026/03_March_2026/1980_Find_Unique_Binary_String.cpp
class Solution {
public:
    string findDifferentBinaryString(vector<string>& bit_pool) {
        string unique_seq = "";
        for (int idx = 0; idx < bit_pool.size(); ++idx) {
            unique_seq += (bit_pool[idx][idx] == '0' ? '1' : '0');
        }
        return unique_seq;
    }
};
